qtfred/src/mission/missionsave.h

```diff
@@ -146,6 +146,20 @@ class CFred_mission_save {
 	 */
 	int save_bitmaps();
 
+	/**
+	 * @brief Saves the campaign file to the given full pathname
+	 *
+	 * @param[in] pathname The full pathname to save to
+	 *
+	 * @details Returns the value of CFred_mission_save::err, which is:
```

Comment:
Missed it on copying, will clarify

Reply:
Actually, at the end of this function, err is returned, because I copied that part over from fred2/missionsave. But it doesn't make sense here or there; saving the campaign never calls save_mission_internal, which seems to be the only place err is set. (I falsely assumed the lower-level output functions would set it). I'll change it to always return 0, unless the file cannot be opened. Right now I can't think of a detectable other way saving can fail.